test/end-to-end/Azure.Functions.NodejsWorker.E2E/Azure.Functions.NodejsWorker.E2E/Fixtures/FunctionAppFixture.cs

```diff
@@ -48,7 +51,6 @@ protected virtual void Dispose(bool disposing)
                     if (_funcProcess != null)
                     {
                         _logger.LogInformation($"Shutting down functions host for {Constants.FunctionAppCollectionName}");
-                        _funcProcess.Kill();
```

Comment:
I think this should be brought back? I think it's causing test executions to run for ~1 hr after tests have run!